File: DebtManagement/src/main/java/vn/diepgia/mchis/DebtManagement/responses/InvoiceResponse.java
package vn.diepgia.mchis.DebtManagement.responses;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import vn.diepgia.mchis.DebtManagement.models.InvoiceLine;

import java.time.LocalDate;
import java.util.List;

@Builder
@NoArgsConstructor
@AllArgsConstructor
@Data
public class InvoiceResponse {

    private String id;
    private LocalDate date;
    private Integer total;
    private List<InvoiceLine> invoiceLines;
    private CustomerResponse customer;
}

File: DebtManagementUI/src/app/api-services/models/invoice-response.ts
/* tslint:disable */
/* eslint-disable */
import { CustomerResponse } from '../models/customer-response';
import { InvoiceLine } from '../models/invoice-line';
export interface InvoiceResponse {
  customer?: CustomerResponse;
  date?: string;
  id?: string;
  invoiceLines?: Array<InvoiceLine>;
  total?: number;
}
